lib/msf/core/exploit/remote/kerberos/ticket.rb

```diff
@@ -7,7 +7,7 @@ class Exploit
     class Remote
       module Kerberos
         module Ticket
-          def create_ticket(enc_key:, start_time:, end_time:, sname:, flags:, domain:, username:, domain_sid:, save_ccache: true)
+          def create_ticket(enc_key:, start_time:, end_time:, sname:, flags:, domain:, username:, user_id:, domain_sid:, save_ccache: true)
```

Comment:
```suggestion
          def create_ticket(enc_key:, start_time:, end_time:, sname:, flags:, domain:, username:, user_id: Rex::Proto::Kerberos::Pac::DEFAULT_ADMIN_RID, domain_sid:, save_ccache: true)
```

And it looks like we'll want to change:

https://github.com/rapid7/metasploit-framework/blob/463915b482991e737e2e5dbdd2b26f4f80312af6/lib/msf/core/exploit/remote/kerberos/client/pac.rb#L47

To be:

```diff
-  user_id = opts[:user_id] || Rex::Proto::Kerberos::Pac::DEFAULT_USER_SID
+  user_id = opts[:user_id] || Rex::Proto::Kerberos::Pac::DEFAULT_USER_RID
```

https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/understand-security-identifiers